openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -21,30 +21,29 @@ <h2 class="step__title">
                         {% endif %}
                     </span>
                 </button>
-            </h2>
+            </h4>
         </header>
 
         <div class="ui-slidable__content" id="oa_grade_{{ xblock_id }}_content" aria-labelledby="oa_grade_{{ xblock_id }}">
             <div class="wrapper--step__content">
                 <div class="step__content">
                     <article class="submission__answer__display step__content__section">
-                        <h3 class="submission__answer__display__title">{% trans "Your Response" %}</h3>
```

Comment:
The header is entirely deleted, but I believe it should be present as an h5. It's present in other variations--

![image](https://cloud.githubusercontent.com/assets/484484/20900464/3594470e-bafc-11e6-8090-79d9ea0e0f92.png)